script/release_utils/check_branch_name.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Get the current branch name
+BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+
+CML_VERSION=$1
+
+if [[ -n "$CML_VERSION" ]]; then
+    EXPECTED_BRANCH_NAME="release/${CML_VERSION:0:3}.x"
```

Comment:
Does this work with something like 10.32.43 ?

Reply:
absolutely not 🙂 good catch